pkg/util/filesystem/util_windows.go

```diff
@@ -0,0 +1,31 @@
+//go:build windows
+// +build windows
+
+/*
+Copyright 2022 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package filesystem
+
+import (
+	"path/filepath"
+)
+
+// IsAbs returns wether the given path is absolute or not.
```

Comment:
same qq as in the original PR. Is this k8s specific? If not, it will be great to raise this as a bug with the `filepath` package to have it resolved for everybody.

Reply:
It shouldn't be k8s specific. Will raise an issue with the ``filepath`` package, but that will take some time to get merged and released, and then adopted by the k8s builders.

Reply:
thank you. I'm not suggesting to wait for this to happen outside k8s, but it will be nice to remove this logic once it's fixed. Would you please update the comment with the link to an issue once created?